src/main/java/duke/util/Ui.java

```diff
@@ -0,0 +1,96 @@
+package duke.util;
+
+import duke.task.Task;
+import duke.task.TaskList;
+
+import java.util.ArrayList;
+
+public class Ui {
+    private final String LINE = "    ____________________________________________________________";
+    private final String INDENT = "     ";
+    private final String NAME = "Alfred";

```

Comment:
your chatbot name is Alfred so ideally your main class and packages name should be changed to Alfred as well.